### PR TITLE
Feature/computer logic

### PR DESCRIPTION
**Pull request by MarrisSparrisNilsson**
Added movement for computer player

### CODE REVIEW

File: ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
     "m_Dictionary": {
+<<<<<<< HEAD
```
Comment (Stocken99) [inline ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json:3]: Merge Conflict!

File: ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+        "m_DictionaryValues": []
+=======
```
Comment (Stocken99) [inline ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json:5]: Merge Conflict!

File: ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json [hunk 3]
```diff
@@ -22,5 +25,6 @@
                 "value": "{\"m_Value\":true}"
             }
         ]
+>>>>>>> main
```
Comment (Stocken99) [inline ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json:28]: Merge Conflict!

File: ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json [hunk 4]
```diff
@@ -22,5 +25,6 @@
...
     }
 }
```
Comment (Stocken99) [inline ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json:30]: No new line at the end of the file?

**Review by Stocken99 — CHANGES_REQUESTED** [review]

**Review by JoelScarinius — CHANGES_REQUESTED** [review]
Look at all your files you have pushed merge conflicts.

**Review by JoelScarinius — APPROVED** [review]
LGTM!

**Review by Stocken99 — APPROVED** [review]

**Pull request merged**